pkg/module/drift/model.go

```diff
@@ -285,3 +292,27 @@ func (rs *Results) process() {
 
 	rs.Text = strings.Join(lines, "\n")
 }
+
+// UniqueSort sorts input alphabetically and returns only non-duplicate entries
+func UniqueSort(input []string) []string {
```

Comment:
feels like this function should be in a different package, maybe in internal package under sort.go